tetrateci/test_1.11.sh

```diff
@@ -64,20 +64,6 @@ for pkg in $PACKAGES; do
     SKIP_TEST_FLAGS+=( "--istio.test.skip=${test}" )
   done
 
-  go test \
```

Comment:
may I know the reason of skip this test? is it related to ARM build or removed for another reason?

Reply:
These are the tests for distroless images. We recently started testing distroless images in e2e test suite. 
https://github.com/tetrateio/tid-internal/issues/51.
Many test cases were failing for integration tests when ran against distroless images in 1.12 and 1.11 like this.
`FAIL: TestSecureNaming/From_primary-0/mTLS_cert_validation_with_plugin_CA`
faililng due to     this error as distroless istio-proxy does not have openssl binary for older releases.
`secure_naming_test.go:139: failed to dump certificate: get cert retry failed with err: the output doesn't contain certificate: OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "openssl": executable file not found in $PATH: unknown`
New builds have echo client functionality which is capable for reading x509 certs params.